src/custom-elements/src/components/date/date.tsx

```diff
@@ -150,7 +150,7 @@ export class Date {
   };
 
   private _initCalendarViewModel(): void {
-    dayjs.locale(this.locale);
+    // dayjs.locale(this.locale);
```

Comment:
@sabah1910 Please uncomment this line and Rebase from master

Reply:
@janibolkvadze Done